src/displayapp/LittleVgl.cpp

```diff
@@ -176,17 +176,24 @@ void LittleVgl::FlushDisplay(const lv_area_t* area, lv_color_t* color_p) {
   lv_disp_flush_ready(&disp_drv);
 }
 
-void LittleVgl::SetNewTouchPoint(uint16_t x, uint16_t y, bool contact) {
+void LittleVgl::SetNewTouchPoint(uint16_t x, uint16_t y, bool contact, bool single_event) {
   tap_x = x;
   tap_y = y;
   tapped = contact;
+  tap_single_event = single_event;
 }
 
 bool LittleVgl::GetTouchPadInfo(lv_indev_data_t* ptr) {
   ptr->point.x = tap_x;
   ptr->point.y = tap_y;
   if (tapped) {
     ptr->state = LV_INDEV_STATE_PR;
+    if (tap_single_event) {
+      // If a tap consists of only a single event, enqueue a synthetic release state update
+      tapped = false;
+      tap_single_event = false;
+      return true;
+    }
```

Comment:
I handled this in https://github.com/InfiniTimeOrg/InfiniTime/pull/1130/commits/2476b34b07b8f834879a25f342127dc9560eb5de , in a new PR at https://github.com/InfiniTimeOrg/InfiniTime/pull/1130 by introducing a new function instead of adding another parameter.